core-java/src/test/java/org/baeldung/java/JavaStringConversionUnitTest.java

```diff
@@ -0,0 +1,189 @@
+package org.baeldung.java;
+
+import org.junit.Test;
+
+import java.io.UnsupportedEncodingException;
+import java.nio.charset.StandardCharsets;
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.util.Arrays;
+import java.util.Date;
+
+/**
+ * Created by sic.org on 8/2/2016.
+ */
+public class JavaStringConversionUnitTest {
+    @Test
+    public void stringToboolean() {
```

Comment:
the name does not follow the naming convention. You can say stringToPrimitiveBoolean. Follow similar norm in other test methods as well.